openslides_backend/action/action.py

```diff
@@ -223,7 +224,7 @@ def check_for_archived_meeting(self, instance: Dict[str, Any]) -> None:
         )
         if not meeting.get("is_active_in_organization_id"):
             raise ActionException(
-                f'Meeting {meeting.get("name", "")}/{meeting_id} cannot be changed, because it is archived.'
+                f'Meeting {meeting_id} ("{meeting.get("name", "")}") cannot be changed, because it is archived.'
```

Comment:
Changed this because `x/y` is typically an fqid and it looked quite confusing ;)